app/reducers/views/i18n.js

```diff
@@ -3,7 +3,7 @@
 
 import {combineReducers} from 'redux';
 
-import Config from 'config';
+import Config from 'config/index';
```

Comment:
cause for some weird reason when running the app it complains by not being able to find "config"